mapsforge-themes/src/main/resources/assets/mapsforge/motorider.xml

```diff
@@ -329,10 +329,10 @@
 	
     <!-- color of normal land unless defined as forest, grass etc -->
     <rule e="way" k="natural" v="nosea">
-        <area fill="#ffffff" stroke="#ffffff" stroke-width="1.0" />
+        <area fill="#f2f2f2" stroke="#f2f2f2" stroke-width="1.0" />
```

Comment:
Does this have to be a shade of light gray and not pure white? We just got rid of the awful gray layer of "neutral hills" and we were like "Oh no, not again!" 😅 Check out the before/after screenshot below:

![image](https://github.com/user-attachments/assets/a23ea7f1-e7be-4558-b7b4-41d05d5e7701)

Reply:
@Sublimis The white brightness is a bit intense for the eyes.
That's why we test the a light gray, useful especially with other map elements.

It was discussed here:
- https://github.com/devemux86/cruiser/discussions/612#discussioncomment-11532643

Reply:
@Sublimis Light theme has again white background via #1594.